### DISCUSSION TITLE
Continuous scanning - Trivy

### DISCUSSION THREAD

**Discussion by javixeneize** [aquasecurity/starboard#1155] [Questions & Help]
Hi

Quick question about trivy. Does it run continuous scanning against the images running in the cluster? If so, how often? Every time the DB is updated?

Thanks

**Comment by nissessenap** [comment]
This is supported through TTL.
You can see how to use it in the helm chart here:
https://github.com/aquasecurity/starboard/blob/3a8075a1ed420494f06b9427557654d3ef1743d8/deploy/helm/values.yaml#L40-L43
There is also support for it in the operator.
I personally run my cluster with a TTL of 24 hours to continuously rescan my images.
